libs/vm-modules/src/math/bignumber.cpp

```diff
@@ -250,6 +259,127 @@ bool UInt256Wrapper::FromJSON(JSONVariant const &variant)
   return true;
 }
 
+void UInt256Wrapper::Add(Ptr<Object> &lhso, Ptr<Object> &rhso)
+{
+  Ptr<UInt256Wrapper> lhs = lhso;
+  Ptr<UInt256Wrapper> rhs = rhso;
+  if (lhs->IsTemporary())
+  {
+    lhs->number_ += rhs->number_;
+    return;
+  }
+  if (rhs->IsTemporary())
+  {
+    rhs->number_ += lhs->number_;
+    lhso = std::move(rhs);
+    return;
+  }
+  Ptr<UInt256Wrapper> n(
+      new UInt256Wrapper(vm_, fetch::vm::TypeIds::UInt256, lhs->number_ + rhs->number_));
+  lhso = std::move(n);
+}
+
+void UInt256Wrapper::Subtract(Ptr<Object> &lhso, Ptr<Object> &rhso)
+{
+  Ptr<UInt256Wrapper> lhs = lhso;
+  Ptr<UInt256Wrapper> rhs = rhso;
+  if (lhs->IsTemporary())
+  {
+    lhs->number_ -= rhs->number_;
+    return;
+  }
+  if (rhs->IsTemporary())
+  {
+    rhs->number_ -= lhs->number_;
+    lhso = std::move(rhs);
+    return;
+  }
+  Ptr<UInt256Wrapper> n(
+      new UInt256Wrapper(vm_, fetch::vm::TypeIds::UInt256, lhs->number_ - rhs->number_));
```

Comment:
same comment as above for typeid in the constructor.

Reply:
Refactored the constructor from underlying container.